src/drm/fairplay.js

```diff
@@ -6,40 +6,41 @@ import {DrmScheme} from './drm-scheme';
 const _logger = getLogger('FairPlay');
 
 const FairPlay: IDrmProtocol = class FairPlay {
+  static scheme: string = DrmScheme.FAIRPLAY;
```

Comment:
so need to add it in others as well (playready, widevine)
but why do we need it at all?

Reply:
i need a way to fetch the  selectd drm scheme in the native adatpter